### PR TITLE
Grants the ability to open Clown slots once more.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title. Skyrat disabled it because they hate fun. We don't.

## Why It's Good For The Game

Say a Clown dies. Is arrested, or is just a bit shit.
What can you do?
Jackshit.
Pray, fax Clown Planet, but why wait on God or the Clown in the High Castle when you can do it yourself?
This also let's Antags, or Clowns summon more friends. Assuming people in the lobby wanna be a clown.

P.S. I commented out and unticked the Skyrat file for double safety. Tried to just override it, didn't work. If you have a better idea for this. You have the floor.

## Changelog

:cl:
add: Clown slots are re-openable by royal decree. The incident between Nanotrasen and King Honkington has been resolved.
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/jobs/job_types/clown.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
-/datum/job/clown
+/*/datum/job/clown //ZUBBERS REMOVAL -- READDING MORE CLOWN SLOTS
 	vox_outfit = /datum/outfit/vox/clown
 	job_flags = STATION_JOB_FLAGS | JOB_CANNOT_OPEN_SLOTS
```
Comment (Majkl-J) [inline modular_skyrat/master_files/code/modules/jobs/job_types/clown.dm:3]: you can just remove the bitflag with

```
/datum/job/clown/New()
	job_flags -= JOB_CANNOT_OPEN_SLOTS
	return ..()
```

**Review by Majkl-J — COMMENTED** [review]
This should work hopefully

File: modular_skyrat/master_files/code/modules/jobs/job_types/clown.dm (hunk 1 above)
Comment (theselfish) [reply modular_skyrat/master_files/code/modules/jobs/job_types/clown.dm:3]: Thank you.

File: modular_zubbers/code/modules/jobs/job_types/clown.dm [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/datum/job/clown/New()
+	job_flags -= JOB_CANNOT_OPEN_SLOTS
+	return ..()
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/jobs/job_types/clown.dm:3]: Ok so I thought about this and this is a terrible way to do it if skyrat ever brings these back.

This however should work
```suggestion
/datum/job/clown/New()
	job_flags = job_flags & (~JOB_CANNOT_OPEN_SLOTS)
	return ..()
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]
We love bitflag operations. Anyhow, approved

File: modular_zubbers/code/modules/jobs/job_types/clown.dm [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/datum/job/clown/New()
+	job_flags = job_flags & (~JOB_CANNOT_OPEN_SLOTS)
```
Comment (Cyprex) [inline modular_zubbers/code/modules/jobs/job_types/clown.dm:2]:
```suggestion
	//remove JOB_CANNOT_OPEN_SLOTS bitflag in a modular fashion.
	job_flags = job_flags & (~JOB_CANNOT_OPEN_SLOTS)
```

**Comment by Cyprex** [comment]
ngl I would prefer this be not done modularly because you can read the base code and it will say JOB_CANNOT_OPEN_SLOTS and it is all just a big fat lie- making code unreadable like this isn't great imo- but I don't think I'm in the majority here

**Comment by nevimer** [comment]
it is fine. not a big deal for a single line.

**Pull request merged**